docs/documentation/server_admin/topics/realms/cross-domain-trust.adoc

```diff
@@ -0,0 +1,21 @@
+[[cross_domain_trust]]
+=== Configuring Cross-Domain Trusts
+
+Cross-domain trusts can be defined at the realm level to specify trusted security domains external to Keycloak. This set of trusted
+domains is used by other features, such as the <<_assertion_grant,JWTs as Authorization Grants>> feature which allows external entities
+to request access tokens for users.
+
+To configure a trusted domain:
+
+. Go to `Realm settings`
+. Select the `Cross-Domain Trust` tab
+. Create a trusted domain entry
+. Save the changes
+
+Each cross-domain trust configuration contains 3 elements:
+
+* Issuer: The URI identifying this trusted domain. This value will be used to verify the issuer on assertions signed by this trusted domain.
+* Audience: The URI identifying the audience of assertions submitted by this trusted domain. This value will be used to verify the audience on assertions signed by this trusted domain.
```

Comment:
What if there are multiple audiences, and different client use different audiences?

As an example let's imagine we're setting up a cross-domain trust with Google as the assertion provider. In a realm there are two clients (client-a and client-b), where each client has their own client configured with Google.